enterprise/frontend/src/embedding-sdk/lib/interactive-question/load-question.ts

```diff
@@ -44,6 +45,9 @@ export const loadQuestionSdk =
       originalCard && new Question(originalCard, metadata);
 
     let question = new Question(card, metadata);
+    if (targetDashboardId) {
+      question = question.setDashboardId(targetDashboardId);
```

Comment:
When setting the dashboard ID. In the save modal, it will hide the collection and dashboard select and show dashboard tab selector when there are more than 1 tabs instead.

If we don't set the dashboard tab, we will see the collection and dashboard selector instead.